frontend/src/components/app-config/user-config-form.tsx

```diff
@@ -343,6 +343,39 @@ export const UserConfigForm: React.FC = () => {
               </FormItem>
             )}
           />
+          <FormField
+            control={form.control}
+            name="runtime.on_cell_change"
+            render={({ field }) => (
+              <div className="flex flex-col gap-y-1">
+                <FormItem className="flex flex-row items-start space-x-2 space-y-0">
+                  <FormControl>
+                    <NativeSelect
+                      data-testid="on-cell-change-select"
+                      onChange={(e) => field.onChange(e.target.value)}
+                      value={field.value}
+                      disabled={isWasm}
```

Comment:
we can remove `disabled={isWasm}` right?

Reply:
good catch, thanks!